content/docs/learning-guide/tutorials/assets/actor-assets.md

```diff
@@ -0,0 +1,92 @@
+
+    Additional mesh groups can be created by choosing **Add another mesh**. However, an actor entity can only have one mesh component. All of the actor's meshes, that is, all the meshes that are bound to the actor's skeleton, must be included in a single mesh group. The actor in this example has one mesh, though it's common for actors to have many unique meshes.
+
+    The following table explains the important elements of the default mesh group for actors:
+
+    | Element | Description |
+    | --- | --- |
+    | **Name mesh** | This property is the name of the mesh group. All product assets of this mesh group use this string as a prefix for their name. The product assets appear in Asset Browser beneath the source asset. By default, the name of the source scene file is used. |
+    | **Select meshes** | The meshes selected for processing as part of the mesh group. Choosing the {{< icon browse-edit-select-files.svg >}} **node select** button allows you to select which meshes from the `.fbx` source asset to include in the mesh group. By default, all meshes in the `.fbx` source asset are selected. |
+    | **Skin** | The [Skin](/docs/user-guide/assets/scene-settings/meshes-tab/#skin) modifier is automatically added for meshes that are bound to a skeleton. The Skin modifier sets the max weights per vertex for the mesh, and the weight threshold for bone influences. The **Max weights per vertex** property has a default value of `8`, so that up to 8 bones can influence each vertex in the mesh. The default **Weight threshold** is `0.001`. Any vertex weighs lower than the threshold value are ignored. |
+    | **Material** | The [Material](/docs/user-guide/assets/scene-settings/meshes-tab/#material) modifier is automatically added to mesh groups. With the Material modifier, you can choose to update the materials for the processed actor, and remove unused materials that might have been previously processed for the actor. |
+
+    {{< note >}}
+If your actor has custom normals and materials that use normal maps, you might need to add [**Mesh (Advanced)**](/docs/user-guide/assets/scene-settings/meshes-tab/#mesh-advanced) and [**Tangents**](/docs/user-guide/assets/scene-settings/meshes-tab/#tangents) modifiers as well. To learn more about mesh modifiers, refer to the [Meshes Tab](/docs/user-guide/assets/scene-settings/meshes-tab) topic in the User Guide.
+    {{< /note >}}
+
+1. Choose the {{< icon browse-edit-select-files.svg >}} file select button to view and select which meshes to include in the actor's mesh group.
+
+    ![ Select meshes for actor. ](/images/learning-guide/tutorials/assets/select-actor-mesh.png)
+
+1. In the **Select nodes** dialog, select only the mesh nodes required by the actor. Mesh nodes are denoted by a mesh icon. Choose **Select** to complete the selection.
```

Comment:
When I opened the fbx from the default pose character that I downloaded from Mixamo, all the bones were selected, in addition to the two meshes it had. I interpreted this "select only mesh nodes" step to mean that I should unselect all these bones, so the action I took was to "unselect all", and then select only the two meshes. Is this common? You might want to mention a little background on this, for the benefit of beginners like me.

Reply:
Bones are ignored by the mesh builder, but animation controls might not be ignored, causing issues. Many users will just mentally lump animation controllers in with bones. I don't want to go into the weeds on this point, though, because this and other node selection issues are actively being addressed. I think the best thing to do is to simply tell users to select only the meshes required by the mesh group.